tensorflow_datasets/image_classification/kaokore/dummy_data_generation.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
```

Comment:
It would be great if you could pull the latest tfds datasets master branch and re-generate the `dataset_info.json` as it now also contains the `module_name` field to help point to the dataset source code in documentation. Thanks!

Reply:
The `module_name` is generated at runtime, It is not contained in the PR, so there is nothing to update.

Reply:
@Conchylicultor I thought we would need the updated `dataset_info.json` uploaded to gist (which was linked in the PR) while adding/updating documentation, won't we? 🤔